core/src/test/scala/cell/base.scala

```diff
@@ -1,15 +1,12 @@
 package cell
 
 import org.scalatest.FunSuite
-
 import java.util.concurrent.CountDownLatch
 
-import scala.util.{ Success, Failure }
-import scala.concurrent.{ Promise, Await }
+import scala.util.{ Failure, Success }
+import scala.concurrent.{ Await, Promise }
 import scala.concurrent.duration._
-
-import lattice.{ Lattice, StringIntLattice, LatticeViolationException, StringIntKey }
-
+import lattice._
```

Comment:
We should normally avoid wildcard imports, unless the number of imported members is getting out of hand.